doc/releases/changelog-dev.md

```diff
@@ -218,6 +218,13 @@
   This is unlikely to affect users since only under certain conditions did
   nesting qnodes worked successfully.
 
+* Changes in the `debug.compile_from_mlir`'s API.
```

Comment:
Wait is this function even still needed? I thought it was supplanted by `replace_ir`

Reply:
Let me look and see if anyone is using it. If not, I will also delete it. :)

Reply:
https://github.com/PennyLaneAI/catalyst/pull/1181/commits/2e723e0b6e2f955503ecb8a963ef1293e9337378

Reply:
Love it!